.buildkite/release-package-registry-distribution.yml

```diff
@@ -1,7 +1,7 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
 env:
-  SETUP_GOLANG_VERSION: '1.20.4'
+  SETUP_GOLANG_VERSION: '1.21.3'
```

Comment:
This should be a leftover.

It should not be needed here. This environment variable is already defined in the `pre-command` hook: https://github.com/elastic/package-registry/blob/3e2cefe386757e5c371720e55230c05fd170c5a5/.buildkite/hooks/pre-command#L9


And I don't see that `golang` is installed neither in the scripts `validate-tag.sh` or `publish-distribution.sh` of this pipeline.

```suggestion
```

Reply:
Removed! Thanks.